metricbeat/module/system/raid/_meta/docs.asciidoc

```diff
@@ -1,3 +1,6 @@
 This is the raid metricset of the module system. It collects stats about the raid.
 
+This metricset is available on:
+- Linux
+
 The config option `raid.mount_point:` can be used to configure the raid mount point.
```

Comment:
Maybe we should add a note here for container users mentioning that `/sys` is needed in the `hostfs` directory if this metricset is used.